js/browser.js

```diff
@@ -66,7 +66,8 @@ var Billing = Class(Emitter, function (supr) {
   this.requestLocalizedPrices = function (products, cb) {
     var res = {}, id;
 
-    if (!fbinstant.payments_ready) {
+    if (!FBInstant.getSupportedAPIs()
+        .includes('payments.purchaseAsync')) {
```

Comment:
when was payments_ready set to true ?

Reply:
after initializeAsync in parallel to the startGame callback, I think nasim is making changes.. i will decline this request